### PR TITLE
fix: add feature name and reference to raw.var

### PR DESCRIPTION
**Pull request by seve**
#203

### CODE REVIEW

**Comment by brianraymor** [comment]
@jahilton should be able to provide you with a test dataset.

File: cellxgene_schema_cli/cellxgene_schema/write_labels.py [hunk 1]
```diff
@@ -290,16 +291,19 @@ def _add_column(self, component: str, column: str, column_definition: dict):
 
             # The sintax below is a programtic way to access obs and var in adata:
             # adata.__dict__["_obs"] is adata.obs
-            self.adata.__dict__["_" + component][new_column_name] = new_column
+            if "." in component:
+                [first_elem, second_elem] = component.split(".")
+                self.adata.__dict__["_" + first_elem].__dict__["_" + second_elem][new_column_name] = new_column
+            else:
+                self.adata.__dict__["_" + component][new_column_name] = new_column
```
Comment (seve) [inline cellxgene_schema_cli/cellxgene_schema/write_labels.py:299]: Could probably functionalize `.__dict__["_" + component]`, but not sure where to store that function

File: cellxgene_schema_cli/cellxgene_schema/write_labels.py [hunk 2]
```diff
@@ -290,16 +290,19 @@ def _add_column(self, component: str, column: str, column_definition: dict):
 
             # The sintax below is a programtic way to access obs and var in adata:
             # adata.__dict__["_obs"] is adata.obs
```
Comment (MDunitz) [inline cellxgene_schema_cli/cellxgene_schema/write_labels.py:292]: can you add a comment about doing the split to support var.raw

**Review by MDunitz — CHANGES_REQUESTED** [review]
lgtm with a small change

**Comment by seve** [comment]
> @jahilton should be able to provide you with a test dataset.

Confirmed working with test dataset!

```
                    feature_type          ensemblid feature_biotype
gene_id
ENSG00000223972  Gene Expression  ENSG00000223972.5            gene
ENSG00000227232  Gene Expression  ENSG00000227232.5            gene
ENSG00000278267  Gene Expression  ENSG00000278267.1            gene
ENSG00000243485  Gene Expression  ENSG00000243485.5            gene
ENSG00000284332  Gene Expression  ENSG00000284332.1            gene
```

```
                    feature_type          ensemblid  ... feature_name feature_reference
gene_id                                              ...
ENSG00000223972  Gene Expression  ENSG00000223972.5  ...      DDX11L1    NCBITaxon:9606
ENSG00000227232  Gene Expression  ENSG00000227232.5  ...       WASH7P    NCBITaxon:9606
ENSG00000278267  Gene Expression  ENSG00000278267.1  ...    MIR6859-1    NCBITaxon:9606
ENSG00000243485  Gene Expression  ENSG00000243485.5  ...  MIR1302-2HG    NCBITaxon:9606
ENSG00000284332  Gene Expression  ENSG00000284332.1  ...    MIR1302-2    NCBITaxon:9606
```

**Review by pablo-gar — APPROVED** [review]
LGTM!

**Pull request merged**